### DISCUSSION TITLE
Yetiforce registration for private users or companies with no online website

### DISCUSSION THREAD

**Discussion by testmodem** [YetiForceCompany/YetiForceCRM#16443] [General] [answered]
Dear concern ..

I am trying to get yetiforce registration completed but it always remain incomplete or declined. I do not work for a company that has an onine portal neither I have a personal website. Can you please advise how to proceed with this ?

![yetiforce-registration-request](https://user-images.githubusercontent.com/59494950/189124702-3063b949-c970-48c6-a894-051d4fa874b5.png)

**Comment by mariuszkrzaczkowski** [comment, marked as answer]
It is best to report problems with registration to registration@yetiforce.com
Send APP ID to this e-mail and I will check it.

**Reply by testmodem** [reply]
Thanks .. much appreciated.
